openlibrary/templates/books/edit.html

```diff
@@ -34,6 +34,7 @@
             note = ""
     <h2 class="editFormTitle">$:title</h2>
     $if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
+        <button type="submit" class="hidden" name="_save" form="addWork" tabindex="-1"></button>
         <span class="adminOnly right"><button type="submit" value="$_('Delete Record')" name="_delete" title="$_('Delete Record')" id="delete" form="addWork">$_("Delete Record")</button></span>
```

Comment:
Reading here: https://tutorial.eyehunts.com/html/html-form-submit-on-enter-key-example-code/ ; can we mark this with `type="button"` instead?